src/components/merge_results/merge-results-helpers.ts

```diff
@@ -0,0 +1,43 @@
+import { Affiliation, Contributor } from '../../types/contributor.types';
+
+const findMatchingContributor = (sourceContributor: Contributor, targetContributors: Contributor[]) => {
+  return sourceContributor.identity.id
+    ? targetContributors.find((c) => c.identity.id === sourceContributor.identity.id)
+    : targetContributors.find((c) => c.identity.name === sourceContributor.identity.name);
+};
+
+const isAffiliationMissing = (sourceAffiliation: Affiliation, targetAffiliations: Affiliation[]) => {
```

Comment:
Tenker at dette er mer en sjekk enn et spørsmål 😅 
```suggestion
const hasMissingAffiliation = (sourceAffiliation: Affiliation, targetAffiliations: Affiliation[]) => {
```